src/app.js

```diff
@@ -6,8 +6,158 @@ import _ from 'underscore';
 import './css/foundation.css';
 import './css/style.css';
 
+import Trip from './app/models/trip';
+import TripList from './app/collections/trip_list';
+
+const TRIP_FIELDS = ['name', 'continent', 'category', 'weeks', 'cost'];
+
+let tripList;
+let tripTemplate;
+let tripDetailsTemplate;
+
 console.log('it loaded!');
 
+const render = function render(tripList) {
+  const tripListElement = $('#trip-list');
+  tripListElement.empty();
+
+  tripList.forEach((trip) => {
+    tripListElement.append(tripTemplate(trip.attributes));
+  });
+
+  $('th.sort').removeClass('current-sort-field');
```

Comment:
What about if it's already sorted by this field?  How could you then reverse the order?